docker-compose.test.yml

```diff
@@ -0,0 +1,11 @@
+version: '2.1'
+
+services:
+  gtsite-service:
+    image: gtsite-service:${GIT_COMMIT}
+    build:
+      context: ./service
+      dockerfile: Dockerfile
+
```

Comment:
For consistency, these services have spacing between them, but the ones in `docker-compose.yml` don't. I think we should try to do the same thing in both.